packages/eslint-config-eslint/default.yml

```diff
@@ -4,6 +4,10 @@ extends:
     - "plugin:node/recommended"
     - "plugin:jsdoc/recommended"
     - "plugin:eslint-comments/recommended"
+    # - "plugin:sonarjs/recommended"
+    - "plugin:unicorn/recommended"
```

Comment:
Can we just enable the rules we want to enable, instead of enabling `unicorn/recommended` and disabling a number of rules we don't want to enable at the moment? I think that would give us much better control over the set of unicorn rules we actually use. In particular, if we enable `unicorn/recommended`, then whenever we upgrade `eslint-plugin-unicorn` we would have to either disable all newly added rules or evaluate whether or not we want to use them. That would be fine if the `unicorn/recommended` rules only catch possible errors but it seems there are many stylistic rules included.

Reply:
I generally prefer to use configs and then opt-out of specific rules that are too aggressive or stylistic, so that we are forced to consider new rules when upgrading versions. Otherwise, it becomes tedious/manual to figure out which rules we are missing out on.

Reply:
I could live with this, and I'm also happy to disable specific stylistic rules the project doesn't want, but I agree with @bmish 's [comment](https://github.com/eslint/eslint/pull/15731#discussion_r836936225) that following the recommended rules tends to force a healthy consideration of new rules of potential benefit. Even the stylistic rules tend to become well-used (and thus more tempting to enforce given that it becomes a de facto standard) when a popular plugin is doing so.

There is a chance the recommended config could drop rules we want, but I haven't seen that happen.

Reply:
I'd be fine with automatically enabling new rules that catch possible errors, but `unicorn/recommended` includes many rules that are not about possible errors. I don't think we should force the team to evaluate each new rule every time we update `eslint-plugin-unicorn` dependency for other reasons such as bug fixes or new options we'd like to use in rules we're already using. We can always evaluate new rules separately.